generator/thorntail-v2/src/main/java/io/fabric8/maven/generator/thorntail/v2/ThorntailV2Generator.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ * <p/>
```

Comment:
Nit-pick: Please keep license format same across all the files. Also, you missed this in ThorntailV2HealthCheckEnricher

Reply:
This is because I copied the WildFly Swarm generator. It has the same problem. Could you recommend a file that has a correct license header? I'll fix both.

Reply:
Sure, look into OpenshiftBuildService or KubernetesClientUtil class.